cpp/src/slice2py/PythonUtil.cpp

```diff
@@ -546,7 +529,11 @@ Slice::Python::ImportVisitor::visitDataMember(const DataMemberPtr& p)
     // For fields with a type that is a Struct, we need to import it as a RuntimeImport, to
     // initialize the field in the constructor. For other contained types, we only need the
     // import for type hints.
-    if (dynamic_pointer_cast<Struct>(type) || dynamic_pointer_cast<Enum>(type))
+    if (auto sequence = dynamic_pointer_cast<Sequence>(type))
```

Comment:
It's not clear to me why we don't have a special import-handling for dictionaries as well. How does it work?